lib/cookies.js

```diff
@@ -91,7 +91,7 @@ Cookies.prototype.set = function(name, value, opts) {
 
   if (opts && signed) {
     if (!this.keys) throw new Error('.keys required for signed cookies');
-    cookie.value = this.keys.sign(cookie.toString())
+    cookie.value = this.keys.sign(cookie.toString()).toString('base64')
```

Comment:
My fault I was looking at the master branch not 1.0.1 branch. In the master branch it returns a buffer instead of a string so I thought that for v1.0.1 to work you needed to make that change.

You can close the pull request.